apps/core/scenes/passengerForm/BaggageBundles/BaggageBundle.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, RadioButton } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BaggageTypeDetail from './BaggageTypeDetail';
+
+type Props = {|
+  +price: ?string,
+  +bags: any,
+  +bagIndex: number,
+  +onToggleCheck: any,
+  +type: string,
+  +currentlyChosen: number,
+|};
+
+type State = {|
+  checked: boolean,
+|};
+
+export default class BuggageBundle extends React.Component<Props, State> {
+  state = {
+    checked: this.props.bagIndex === 0,
```

Comment:
Can't we try to do it without setting `props` in initial state? As far as I know it's bad practise (https://stackoverflow.com/questions/27376725/react-getinitialstate-using-props)

Reply:
Oh my bad 😞Will change it

Reply:
Much better now 👍